privaterelay/settings.py

```diff
@@ -379,6 +379,7 @@ def _get_initial_middleware() -> list[str]:
 SUBSCRIPTIONS_WITH_VPN = config("SUBSCRIPTIONS_WITH_VPN", default="", cast=Csv())
 
 MAX_ONBOARDING_AVAILABLE = config("MAX_ONBOARDING_AVAILABLE", 0, cast=int)
+MAX_ONBOARDING_FREE_AVAILABLE = config("MAX_ONBOARDING_AVAILABLE", 0, cast=int)
```

Comment:
Did you mean to reuse `MAX_ONBOARDING_AVAILABLE`, or should this be:
```suggestion
MAX_ONBOARDING_FREE_AVAILABLE = config("MAX_ONBOARDING_FREE_AVAILABLE", 0, cast=int)
```